sdk/src/main/java/io/horizen/account/state/EvmMessageProcessor.java

```diff
@@ -10,10 +9,14 @@
 import java.math.BigInteger;
 
 public class EvmMessageProcessor implements MessageProcessor {
-    protected Address[] nativeContractAddresses = new Address[] {
-        WellKnownAddresses.WITHDRAWAL_REQ_SMART_CONTRACT_ADDRESS(),
-        WellKnownAddresses.FORGER_STAKE_SMART_CONTRACT_ADDRESS(),
-    };
+    protected Address[] nativeContractAddresses = null;
```

Comment:
I think we can make this variable private just to avoid for future developments that anyone in this package will use the not-initialize addresses list

Reply:
Done